### ISSUE TITLE
Unknown database `mysql` when cluster initialize

### ISSUE THREAD

**Issue by UncP** [pingcap/tidb#6287]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.

`start the cluster and insert some data`

2. What did you expect to see?

3. What did you see instead?

`failed to connect tidb`

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
`TiDB: v2.0.0-rc.4-61-g533fac1`
`TiKV: 2.0.0-rc.4.1`
`PD: v2.0.0-rc.4-5-gc729570`

Here is log, it's pretty short.
[pd.log](https://github.com/pingcap/tidb/files/1910404/pd.log)
[tidb.log](https://github.com/pingcap/tidb/files/1910405/tidb.log)
[tikv.log](https://github.com/pingcap/tidb/files/1910407/tikv.log)

**Comment by UncP** [comment]
@zimulala

**Comment by jackysp** [comment]
What kind of data did you insert? @UncP

**Comment by jackysp** [comment]
I've checked the log file. It looks like the meta data key conflict when parallel alloc table id for the system tables. PTAL @zimulala

**Comment by UncP** [comment]
Just some bank account, `id` as primary key and `value`, part of the step of Jepsen test.

**Comment by zimulala** [comment]
@UncP 
Could give the full error message when the client connection failed? E.g. `[dial tcp 127.0.0.1:4001: connect: connection refused]`

**Comment by UncP** [comment]
<img width="1323" alt="2018-04-16 10 56 34" src="https://user-images.githubusercontent.com/16263931/38788326-aa9a1c5e-4165-11e8-99ae-da81efeb72c1.png">

Jepsen reported it.

**Comment by jackysp** [comment]
I can reproduce you error. Did you start two tidb at the same time for a new cluster? @UncP

**Comment by jackysp** [comment]
```
goroutine 1 [running]:
runtime/debug.Stack(0x2ccc6c8, 0xc42016bd08, 0x100e348)
        /usr/local/Cellar/go/1.10.1/libexec/src/runtime/debug/stack.go:24 +0xa7
runtime/debug.PrintStack()
        /usr/local/Cellar/go/1.10.1/libexec/src/runtime/debug/stack.go:16 +0x22
github.com/pingcap/tidb/session.mustExecute(0x1f8f680, 0xc4202a60d0, 0x1eb310c, 0x93)
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/session/bootstrap.go:709 +0xb9
github.com/pingcap/tidb/session.doDDLWorks(0x1f8f680, 0xc4202a60d0)
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/session/bootstrap.go:643 +0x1b0
github.com/pingcap/tidb/session.bootstrap(0x1f8f680, 0xc4202a60d0)
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/session/bootstrap.go:212 +0xcc
github.com/pingcap/tidb/session.runInBootstrapSession(0x1f75380, 0xc42035dc00, 0x1ebc780)
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/session/session.go:1157 +0xe3
github.com/pingcap/tidb/session.BootstrapSession(0x1f75380, 0xc42035dc00, 0x1f75380, 0xc42035dc00, 0x0)
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/session/session.go:1104 +0x33c
main.createStoreAndDomain()
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/tidb-server/main.go:163 +0x13d
main.main()
        /Users/yusp/work/goprojects/src/github.com/pingcap/tidb/tidb-server/main.go:140 +0x66
```
The unrecovered panic could be this. @UncP @zimulala

**Comment by UncP** [comment]
Jepsen works like this, start 5 node cluster, each node has tidb-tikv-pd, then do test.
What do you mean `start two tidb at the same time for a new cluster`?

It waited 60s to start the test and.

Jepsen does the same test in the same way everyday using the latest release, for the past few weeks, the test is fine, but this error occurs everyday since `v2.0.0-rc.4-61-g533fac1`. Maybe it's related to some commit around `g533fac1`?

**Comment by zimulala** [comment]
@UncP 
Thanks for your report!
I can reproduce your error.  I'll handle it.

**Comment by UncP** [comment]
Thx.

**Comment by winkyao** [comment]
@UncP This issue is fixed in #6286 , please try latest TiDB master.
